src/app/static/src/app/types.ts

```diff
@@ -0,0 +1,39 @@
+//TODO: These should be generated from the mintr schema
```

Comment:
Actually I think these should be used from the dynamic-form package - you should just be able to import them?